rpcserver.go

```diff
@@ -4029,6 +4029,7 @@ func (r *rpcServer) GetNetworkInfo(ctx context.Context,
 		MaxChannelSize:       int64(maxChannelSize),
 		MedianChannelSizeSat: int64(medianChanSize),
 		NumZombieChans:       numZombies,
+		SyncedToGraph:        r.server.authGossiper.SyncManager().IsGraphSynced(),
```

Comment:
If you make the return value channel, then we don't need to block here and can fall through to `false` in that case.

Reply:
Went with a different approach instead.